app/notifications/new_logbook_entry_created_notification.rb

```diff
@@ -0,0 +1,32 @@
+class NewLogbookEntryCreatedNotification < Noticed::Base
+  deliver_by :database, if: :database_notifications?
+  deliver_by :email, mailer: "LogbookEntryMailer", method: "created_email", if: :email_notifications?
+
+  DEFAULT_SEND_NOTIFICATION = true
+  DEFAULT_SEND_EMAIL_NOTIFICATION = true
+
+  param :logbook_entry
+
+  def message
+    aircraft_model = Aircraft.find_by(id: params[:logbook_entry].aircraft_id).model
+    pilot_in_command = User.find_by(id: params[:logbook_entry]).first_name
+    "#{pilot_in_command} has created a logbook entry for their flight sim on the #{aircraft_model}."
```

Comment:
Same comments as above. We don't need to be using `.find_by` because the `belongs_to` in the `logbook_entry` model will hook that up for us.